### PR TITLE
Sort strings in translation bundle in alphabet order

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

File: g11n-ws/modules/md-data-api-bundleimpl/src/main/java/com/vmware/vip/messages/data/dao/impl/OneComponentDao.java [hunk 1]
```diff
@@ -42,29 +39,17 @@
...
-		} catch (JsonMappingException e) {
-			String errorLog = ConstantsKeys.FATA_ERROR + e.getMessage();
-			logger.error(errorLog, e);
-			throw new BundleException("File is not existing: ");
-		} catch (IOException e) {
+			result = JSON.parseObject(jsonStr, ResultI18Message.class);
+		} catch (Exception e) {
 			String errorLog = ConstantsKeys.FATA_ERROR + e.getMessage();
 			logger.error(errorLog, e);
 			throw new BundleException("File is not existing: ");
```
Comment (Xiaochao8) [inline g11n-ws/modules/md-data-api-bundleimpl/src/main/java/com/vmware/vip/messages/data/dao/impl/OneComponentDao.java]: Use period instead colon.

File: g11n-ws/modules/md-restful-synch/build.gradle [hunk 2]
```diff
@@ -46,9 +46,8 @@ dependencies {
     compile project(":md-restful-api")
     compile project(":md-service-i18n-l3")
     compile("org.apache.httpcomponents:httpclient:4.5.6")
-    compile("com.alibaba:fastjson:1.2.51")
-    compile("io.springfox:springfox-swagger-ui:2.5.0")
-    compile("io.springfox:springfox-swagger2:2.5.0")
+    compile("io.springfox:springfox-swagger-ui:2.9.2")
+    compile("io.springfox:springfox-swagger2:2.9.2")
```
Comment (Xiaochao8) [inline g11n-ws/modules/md-restful-synch/build.gradle]: Why change swagger-ui? Please create a new PR for it.

File: g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/JSONUtil.java [hunk 3]
```diff
@@ -26,29 +20,18 @@
...
-        ContainerFactory containerFactory = getContainerFactory();
+
         Map<String, Object> result = null;
         try {
-            result = (Map<String, Object>) parser.parse(json, containerFactory);
-        } catch (ParseException e) {
+            result =  JSONObject.parseObject(json, Map.class);
+        } catch (Exception e) {
+        	System.out.println(json);
             e.printStackTrace();
```
Comment (Xiaochao8) [inline g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/JSONUtil.java]: Please use logger to log this.

File: g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/JSONUtil.java [hunk 4]
```diff
@@ -100,9 +83,9 @@ public int compare(String o1, String o2) {
 			}
 		});
 		try {
-			Map<String, Object> genreJsonObject = (Map<String, Object>) JSONValue.parseWithException(jsonStr);
+			Map<String, Object> genreJsonObject = (Map<String, Object>) JSONObject.parseObject(jsonStr, TreeMap.class);
 			sortMap.putAll(genreJsonObject);
-		} catch (ParseException e) {
+		} catch (Exception e) {
 			e.printStackTrace();
```
Comment (Xiaochao8) [inline g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/JSONUtil.java]: same as above

File: g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/translation/BundlesGenerator.java [hunk 5]
```diff
@@ -62,7 +61,7 @@ public void handleRemoteRusult(String remoteRusult) {
                                     + TranslationUtil.genernateJsonLocalizedFileName(tLocale),
                             component, tLocale, (JSONObject) bundleObj.get(tLocale));
                 }
-            } catch (ParseException e) {
+            } catch (RuntimeException e) {
                 e.printStackTrace();
```
Comment (Xiaochao8) [inline g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/translation/BundlesGenerator.java]: Please check all of this kind of issues.

File: g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/JSONUtils.java [hunk 6]
```diff
@@ -50,8 +44,8 @@
 	public static JSONObject string2JSON(String jsonStr) {
 		JSONObject genreJsonObject = null;
 		try {
-			genreJsonObject = (JSONObject) JSONValue.parseWithException(jsonStr);
-		} catch (ParseException e) {
+			genreJsonObject = JSONObject.parseObject(jsonStr);
+		} catch (Exception e) {
```
Comment (Xiaochao8) [inline g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/JSONUtils.java]: Why catch all the exceptions?

File: g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/JSONUtil.java (hunk 3 above)
Comment (tigershi) [reply g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/JSONUtil.java]: this model can't use log4j

File: g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/translation/BundlesGenerator.java (hunk 5 above)
Comment (tigershi) [reply g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/translation/BundlesGenerator.java]: this model can't use log4j

File: g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/JSONUtil.java (hunk 4 above)
Comment (tigershi) [reply g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/JSONUtil.java]: this module can't use logger4j

File: g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/JSONUtils.java (hunk 6 above)
Comment (tigershi) [reply g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/JSONUtils.java]: this change the JSONObject class

File: g11n-ws/modules/md-restful-synch/build.gradle (hunk 2 above)
Comment (tigershi) [reply g11n-ws/modules/md-restful-synch/build.gradle]: I will revert it

**Pull request closed without merging**